package.json

```diff
@@ -29,17 +29,20 @@
   },
   "homepage": "https://github.com/alansouzati/gulp-jest",
   "dependencies": {
-    "jest-cli": "^17.0.0",
     "gulp-util": "^3.0.0",
     "through2": "^2.0.1"
   },
   "devDependencies": {
     "babel-cli": "^6.18.0",
-    "babel-jest": "^17.0.0",
+    "babel-jest": ">=17.0.0",
     "babel-plugin-transform-object-rest-spread": "^6.3.13",
     "babel-preset-es2015": "^6.1.18",
     "babel-preset-react": "^6.1.18",
-    "gulp": "^3.8.7"
+    "gulp": "^3.8.7",
+    "jest-cli": ">=17.0.0"
```

Comment:
It's declared in devDependencies because without it the tests would fail. 😉

And again, `=>` means any version 17 or above is good. In the unlikely case where Jest introduces a breaking change to its public CLI interface, you can publish a patch release with an upper bound e.g. `=>17.0.0 < 19.5.0`, fix the issue, then release a major version with a new lower bound. Hope this makes sense.